apps/condo/domains/news/schema/GetNewsItemsRecipientsCountersService.test.js

```diff
@@ -22,10 +22,13 @@ const { buildPropertyMap } = require('@condo/domains/property/utils/testSchema/f
 const { createTestResident } = require('@condo/domains/resident/utils/testSchema')
 const { makeClientWithNewRegisteredAndLoggedInUser, createTestUser } = require('@condo/domains/user/utils/testSchema')
 
+
 let adminClient, staffClientYes
 let dummyO10n
 describe('GetNewsItemsRecipientsCountersService', () => {
-    beforeAll(async () => {
+    jest.setTimeout(60000)
```

Comment:
It's better to set previous value after this test.

Reply:
What is the previous value? :) Where can I get it?

Reply:
To run tests locally there is a limit of 60 seconds, and in CI it is 15. Therefore it is not clear to which value we should set

Reply:
I'm afraid that this value will set for all not started tests everywhere since this line being executed (in current process). If not, it's ok.

Reply:
Maybe it's possible to use something like that: https://jestjs.io/docs/api#testname-fn-timeout (3rd argument as timeout)

Reply:
I didn't know about this, thanks)
Increased timeout for one test only